### ISSUE TITLE
\added{\footnote{}} doesn't show up with \trackchanges

### ISSUE THREAD

**Issue by mtbannister** [AASJournals/AASTeX60#31]
A use of \added{} that contains a \footnote shows the added text in the correct (red) colour, but the footnote does not show up in the List of Changes generated at the end of the document.

**Comment by gregschwarz** [comment]
This has been fixed in v6.1 which will be release in the coming weeks.

**Comment by augustfly** [comment]
This is fixed (sort of) in v6.1. The footnote does appear added to the text and to the list of changes at the end of the document, however, the footnote \# increments from the last footnote in the main text. In other words, in the attached test code, the list of changes lists the footnote as the 4th footnote, when it is actually the 2nd. The page is correct however, and for the purposes of helping people find changes probably works well enough.  

I would vote this closed, but will await @mtbannister to review. 

``` latex
% test for footnotes.
\documentclass[modern, trackchanges]{aastex61}
\begin{document}
\title{Demo}
\author{Me}
\begin{abstract} Hello \end{abstract}

\section{testing}

\footnote{first footnote!}
\newpage

\added{\footnote{check out this new footnote! It should be the 2nd.}}
\newpage

\footnote{second footnote}

\clearpage

\listofchanges

\end{document}
```

**Comment by mtbannister** [comment]
I agree that as long as it shows in the list as being on the correct page, that will do nicely for allowing people to find the correct footnote. Thanks!
